src/dpos/views/ValidatorList.vue

```diff
@@ -34,12 +34,12 @@
                 </div>
                 <div class="stakes">
                   <label>Stake</label>
-                  <span>{{validator.delegationTotal | tokenAmount}}</span>
+                  <span>{{ validator.delegationTotal }}</span>
```

Comment:
@napongk could you add some css to align amounts and fees to the right

Reply:
ok